### PR TITLE
Update imports, use nogil version of sqrt

### PR DESCRIPTION
**Pull request by jbrockmendel**

### CODE REVIEW

**Comment by jreback** [comment]
would it not be better to expose this as an definition in a single .pxd, maybe call it cmath.pxd (along with other ones), then simply import centrally from there (or could also just add to util.pxd).

**Comment by jbrockmendel** [comment]
> would it not be better to expose this as an definition in a single .pxd, maybe call it cmath.pxd (along with other ones), then simply import centrally from there (or could also just add to util.pxd).

I see the upside to this but also like transparent/direct imports.  Not sure there is enough content to justify a cmath.pxd; what else would go in there?

I'll open an issue with cython to see if they can fix the declaration on their end too and make this (eventually) a non-issue.

For purposes of expediency, would it be easier if I revert the sqrt bits now and get the easy other fixups closed down?

**Comment by jreback** [comment]
i would put it in util.pxd

the issue is it’s very likely that future changes will accidentally just use the wrong import and just do it with d

File: pandas/_libs/window.pyx [hunk 1]
```diff
@@ -32,11 +32,10 @@ cdef double NaN = <double> np.NaN
 cdef inline int int_max(int a, int b): return a if a >= b else b
 cdef inline int int_min(int a, int b): return a if a <= b else b
 
-from util cimport numeric
+from util cimport numeric, sqrt
 
 cdef extern from "../src/headers/math.h":
     int signbit(double) nogil
```
Comment (jreback) [inline pandas/_libs/window.pyx]: prob worthwhile adding any of the imports from src/headers/math here as well (prob not very many)

Comment (jbrockmendel) [reply pandas/_libs/window.pyx]: > prob worthwhile adding any of the imports from src/headers/math here as well (prob not very many)

Agreed; its been added to the src todo list.  Need to move on for now.

**Comment by jbrockmendel** [comment]
conda HTTPError 502 on Appveyor.  Will wait until late-evening so re-push as the build queue is usually short then.

**Comment by jreback** [comment]
looks fine. prefer that you update all of these in this pr.

**Comment by jbrockmendel** [comment]
Travis error in 3.6 TestDataFramePlots.test_parallel_coordinates_with_sorted_labels

**Comment by jbrockmendel** [comment]
> prefer that you update all of these in this pr.

I opened an issue on the cython tracker and based on [the response](https://github.com/cython/cython/issues/2029#issuecomment-348006128) I think we should stick a pin in that until we have a better idea what's going on.

**Comment by jreback** [comment]
are these in cython 0.26 or greater?

**Comment by jbrockmendel** [comment]
> are these in cython 0.26 or greater?

All my local builds should be on 0.27.1

**Comment by jreback** [comment]
and what is the perf affect of this
specifically related to the revert of the nogil sqrt changes

**Comment by jbrockmendel** [comment]
> specifically related to the revert of the nogil sqrt changes

The "revert" is reverting from an earlier version of the PR, i.e. keeps the status quo in master unchanged.  This should be nothing but import cleanup.

**Comment by jreback** [comment]
thanks!

**Pull request merged**